<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProcParmDialog</class>
 <widget class="QDialog" name="ProcParmDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>335</width>
    <height>345</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>360</width>
    <height>345</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Processing Parameters</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Band Pass Filter</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_14">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="myrc.qrc">:/images/Filt.png</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>F1</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="edF1"/>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="font">
             <font>
              <pointsize>8</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Hz</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>F2</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="edF2"/>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="font">
             <font>
              <pointsize>8</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Hz</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>F3</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="edF3"/>
          </item>
          <item>
           <widget class="QLabel" name="label_7">
            <property name="font">
             <font>
              <pointsize>8</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Hz</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_8">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>F4</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="edF4"/>
          </item>
          <item>
           <widget class="QLabel" name="label_9">
            <property name="font">
             <font>
              <pointsize>8</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Hz</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCheckBox" name="ckFilt">
          <property name="text">
           <string>Use Filter</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Automatic Gain Control</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_15">
      <item>
       <widget class="QLabel" name="label_10">
        <property name="font">
         <font>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Window Length</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edAgcw">
        <property name="minimumSize">
         <size>
          <width>72</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_11">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>ms</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>2</number>
        </property>
        <item>
         <widget class="QCheckBox" name="ckAgc">
          <property name="text">
           <string>Use AGC</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="ckNorm">
          <property name="text">
           <string>Use Normalization</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QPushButton" name="okButton">
          <property name="text">
           <string>OK</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="applyButton">
          <property name="text">
           <string>Apply</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="closeButton">
          <property name="text">
           <string>Close</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="myrc.qrc"/>
 </resources>
 <connections/>
</ui>
